tasks/build.py

```diff
@@ -474,6 +475,12 @@ def prepare_jobs(pr, cfg, event_info, action_filter):
     #      call to just before download_pr
     year_month, pr_id, run_dir = create_pr_dir(pr, cfg, event_info)
 
+    accelerator = "none"
```

Comment:
Why use `"none"` instead of `None` here?

Reply:
We could use `None`. Just need to change the processing where it is used as a string and we want a meaningful output, e.g., in a log or in a PR comment.

Reply:
Implemented in https://github.com/EESSI/eessi-bot-software-layer/pull/280/commits/8f44ba9846c7d9eb37fefd232e0884ab9651e627